### DISCUSSION TITLE
pdf解析为markdown后一些字符丢失的问题

### DISCUSSION THREAD

**Discussion by balalaxiaomoxiansoga** [magicpdf/Magic-PDF#828] [Q&A]
您好，我想问一下pdf解析为markdown后一些字符丢失的问题，主要是fi,fl一类

![企业微信截图_17304263154863](https://github.com/user-attachments/assets/5746cf33-8335-4620-a684-11e3b5ec3e55)

**Comment by myhloli** [comment]
能上传一份pdf文件吗

**Reply by balalaxiaomoxiansoga** [reply]
[powell2009.pdf](https://github.com/user-attachments/files/17614379/powell2009.pdf)

**Comment by myhloli** [comment]
这个问题昨天刚修复，你提供的这篇pdf我验证了下应该已经完全没有问题了，您可以在https://www.modelscope.cn/studios/OpenDataLab/MinerU 上自行测试并等待我们的下一次发版(大概本周完成

**Reply by balalaxiaomoxiansoga** [reply]
好的，谢谢

**Reply by myhloli** [reply]
0.9.1 fixed
